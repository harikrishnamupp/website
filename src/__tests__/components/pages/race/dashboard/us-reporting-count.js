import React from 'react'
import renderer from 'react-test-renderer'
import UsReportingCount from '~components/pages/race/dashboard/us-reporting-count'

describe('Components : Pages : Race : Dashboard : US Overview', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <UsReportingCount
          statesCasesCount={49}
          statesDeathsCount={48}
          className="css-class-name"
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
