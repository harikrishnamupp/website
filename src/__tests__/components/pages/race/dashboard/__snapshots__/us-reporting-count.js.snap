// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Race : Dashboard : US Overview renders correctly 1`] = `
<div
  className="css-class-name"
>
  <div
    className="data"
  >
    <div
      className="dataItem"
    >
      <h3
        className="title"
      >
        Race/ethnicity data for cases:
      </h3>
      <p>
        <span
          className="percent"
        >
          49
        </span>
         
        <br />
        <span
          className="percentCaption"
        >
          of 56 states & territories
        </span>
      </p>
    </div>
    <div
      className="dataItem"
    >
      <h3
        className="title"
      >
        Race/ethnicity for deaths:
      </h3>
      <p>
        <span
          className="percent"
        >
          48
           
        </span>
        <br />
        <span
          className="percentCaption"
        >
          of 56 states & territories
        </span>
      </p>
    </div>
  </div>
</div>
`;
